RELEASE_NOTES

```diff
@@ -1,6 +1,7 @@
 3.4.0
   - New API
     - StatementException.getShortMessage
+    - Added the MapMappers JdbiConfig class to configure column name case changes in favor of the old boolean toggle.
```

Comment:
> in favor of the old boolean toggle

"In favor of" is the thing you're moving toward, not away from.

Maybe something like:

```
Added MapMappers config class with `caseChange` (or whatever we name it) property, as a replacement for the `MapMapper(boolean)` constructor.